makefile

```diff
@@ -92,5 +93,14 @@ tmp/input/mondo.sssom.tsv: | tmp/input/
 	wget http://purl.obolibrary.org/obo/mondo/mappings/mondo.sssom.tsv -O $@
 
 # creates more than just this file; that goal creates multiple files
-output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: | output/
+output/medgen_terms_mapping_status.tsv output/obsoleted_medgen_terms_in_mondo.txt: tmp/input/mondo.sssom.tsv | output/
 	python src/mondo_mapping_status.py
+
+# ----------------------------------------
+# Robot templates
+# ----------------------------------------
+ftp.ncbi.nlm.nih.gov/pub/meedgen/MedGenIDMappings.txt: ftp.ncbi.nlm.nih.gov
+
+# todo: Ideally I wanted this done at the end of the ingest, permuting from medgen.sssom.tsv, but there were some problems with that file. Eventually changing to that feels like it makes more sense. Will have already been pre-curated by disease. And some of the logic in this Python script is duplicative.
+ROBOT_addMedGen_fromIngest.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
```

Comment:
```suggestion
medgen-mappings.robot.tsv: ftp.ncbi.nlm.nih.gov/pub/medgen/MedGenIDMappings.txt
```

Reply:
Will do. @matentzn You said on Slakck `medgen-xrefs` though. Which do you prefer?

BTW the reason I went with `ROBOT_` is because that is the convention in the `mondo` repo, so when I download and put in the `templates/` folder, that's what I'll call it. Do you want me to:
(a) use medgen-mappings/xrefs.robot.tsv as the name in both this repo and `mondo`?
(b) call it medgen-mappings/xrefs.robot.tsv here, but in `mondo`, call it ROBOT_medgen-mappings/xref.tsv?

Reply:
I went with (b), and stuck with the name `medgen-xrefs.robot.tsv` for this repo.